erp-data-exchange-specification.md

```diff
@@ -280,15 +287,39 @@ The `userRelations.csv` file describes the relation between an agent and a resou
 | **resourceType** | [Resource Type](#resource-type) | The type of the resource being referenced by the resourceId (`property`)                    |
 | **validFromDate** | [DateTime](#datetime-type)  | Optional start date of the validity of the relation (if provided **validToDate** is required as well) |
 | **validToDate**   | [DateTime](#datetime-type)  | Optional end date of the validity of the relation (if provided **validFromDate** is required as well) |
+| **jobRole**   | [string](#string-type)  | Optional |
```

Comment:
Should this be an enum rather than string type?

Reply:
unfortunately not, since the customer can define their jobroles by themselves. Afaik we agreed on not generalizing + translating them